### ISSUE TITLE
Ambiguous implicit conversion with FHIR R4

### ISSUE THREAD

**Issue by brynrhodes** [cqframework/clinical_quality_language#445]
The following library:

```
library Query version '1'
using FHIR version '4.0.0'
include FHIRHelpers version '4.0.0' called FHIRHelpers

context Patient

define TestMedicationOrders:
  ["MedicationRequest"] M
	  where M.doNotPerform is true
```

Produces the error "Ambiguous implicit conversion between System.Boolean and System.Boolean"

**Comment by brynrhodes** [comment]
The issue is that several of the implicit conversions in the FHIR model info are declared multiple times, and the conversion map internally isn't checking for duplicate conversions correctly, so it shows up as ambiguous conversion targets.

**Comment by brynrhodes** [comment]
This issue needs to be applied to the v14 branch as well, as soon as that is merged to master

**Comment by brynrhodes** [comment]
Addressed in v1.3.18-SNAPSHOT
